.github/workflows/action.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         run: msbuild -t:rebuild -property:Configuration=TestWindows
 
       - name: test
-        uses: microsoft/vstest-action@v1.0.0
+        uses: rusty-bender/vstest-action@main
```

Comment:
Can you please rebase? The vstest action was changed because the Microsoft one is no longer functional...

Reply:
Rebased fork over upstream, rebased fix branch over main